src/layer_builder.rs

```diff
@@ -230,7 +228,7 @@ where
         let morton_tile_index = morton_index(tile_pos);
         if morton_tile_index < self.tiles.capacity() {
             if let Some(tile) = &self.tiles[morton_tile_index].1 {
-                return Ok(&tile);
+                return Ok(tile);
```

Comment:
compiler can do this implicitly. I personally don't mind it so I checked with clippy and it complains about this so this updates it to version clippy prefers.